Solutions/SeraphicSecurity/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "seraphicalgorithmsltd1616061090462",
+	"offerId": "seraphic-security-sentinel",
+	"firstPublishDate": "2023-07-31",
+	"lastPublishDate": "2023-07-31",
+	"providers": ["Seraphic Security"],
+	"categories": {
+		"domains" : ["Security"]
```

Comment:
"Domain" change is not updated in SolutionMetadata file, please update it.

Reply:
fixed